src/Whoops/Handler/XmlResponseHandler.php

```diff
@@ -67,7 +67,7 @@ public function contentType()
      */
     private static function addDataToNode(\SimpleXMLElement $node, $data)
     {
-        assert(is_array($data) || $node instanceof Traversable);
+        assert(is_array($data) || $data instanceof Traversable);
```

Comment:
hmm there was a typo in this line... not related to php7.2 but found while working on this line, because of deprecation warnings.